test_module.py

```
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+"""A test module with various utilities."""
+
+
+def utility_function():
```

Ruff (0.11.9)

Missing return type annotation for public function `utility_function`

Add return type annotation: `str`

(ANN201)